### ISSUE TITLE
Cannot run on Docker (after recent release?)

### ISSUE THREAD

**Issue by virginijus-servicebridge** [FusionAuth/fusionauth-issues#870]
Just recently tried to install FusionAuth on Docker and it cannot start.
Here is console output:
fusionauth_1  | You are using a deprecated configuration property of [fusionauth-app.cookie-same-site-policy] in [fusionauth.properties]. The new variable name is [fusionauth-app.http.cookie.same-site-policy]
fusionauth_1  | /usr/local/fusionauth/fusionauth-app/apache-tomcat/bin/setenv.sh: line 91: : invalid variable name

**Comment by robotdan** [comment]
I am unable to recreate, is this your own docker image, or ours? 

```
> docker run -p 9011:9011 -it fusionauth/fusionauth-app:latest     
tty: ignoring all arguments
/dev/pts/0
You are using a deprecated configuration property of [fusionauth-app.cookie-same-site-policy] in [fusionauth.properties]. The new variable name is [fusionauth-app.http.cookie.same-site-policy]
Using CATALINA_BASE:   /usr/local/fusionauth/fusionauth-app/apache-tomcat
Using CATALINA_HOME:   /usr/local/fusionauth/fusionauth-app/apache-tomcat
Using CATALINA_TMPDIR: /usr/local/fusionauth/fusionauth-app/apache-tomcat/temp
Using JRE_HOME:        /opt/openjdk/
Using CLASSPATH:       /usr/local/fusionauth/fusionauth-app/apache-tomcat/bin/bootstrap.jar:/usr/local/fusionauth/fusionauth-app/apache-tomcat/bin/tomcat-juli.jar
...
```

**Comment by virginijus-servicebridge** [comment]
I've run these powershell commands
curl -o docker-compose.yml https://raw.githubusercontent.com/FusionAuth/fusionauth-containers/master/docker/fusionauth/docker-compose.yml && curl -o .env https://raw.githubusercontent.com/FusionAuth/fusionauth-containers/master/docker/fusionauth/.env && docker-compose up

as described in your online https://fusionauth.io/download section

**Comment by robotdan** [comment]
Ha.. ok, I'm an idiot. I was running those same commands, but I forgot to "pull" the latest image so my `:latest` was not the latest. 

I can recreate now.

**Comment by robotdan** [comment]
Release in progress.

**Comment by robotdan** [comment]
Available in 1.19.6
